mithril-common/src/chain_observer/pallas_observer.rs

```diff
@@ -69,51 +68,66 @@ impl PallasChainObserver {
             .acquire(None)
             .await
             .map_err(|err| anyhow!(err))
-            .with_context(|| "PallasChainObserver Failed to acquire statequery")?;
+            .with_context(|| "PallasChainObserver failed to acquire statequery")?;
```

Comment:
I think that for static message errors is more efficient to use `context(str)` instead of `with_context(fn)`

Reply:
the previous implementation was using `context` but it was requested to use `with_context`.

It seems to be an ADR from the mithril team, here: https://mithril.network/doc/adr/6